dev/user.clj

```diff
@@ -15,22 +15,35 @@
    [clojure.string :as string]
    [com.grzm.component.pedestal :as pedestal-component]
    [com.stuartsierra.component :as component]
-   [com.stuartsierra.component.repl :refer [set-init]]
+   [com.stuartsierra.component.repl :refer [reset set-init]]
+   [formatting-stack.component]
    [modular.postgres]
+   [playground.server]
    #_ [background-processing.background-processor :as background-processor]
    #_ [background-processing.enqueuer :as enqueuer]
-   [playground.server]
    [playground.service]))
 
+;; ugly hack, will disappear when we use the newer project template
+(def vemv? (-> (System/getenv "USER") #{"vemv" "victor.valenzuela"}))
```

Comment:
How should I merge and pull the user part, here and in the modular configuration? should I just pull and change locally?

Reply:
`git pull` and that's it

it will work for you, given the `if`s

Reply:
good